code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -106,6 +106,8 @@
 	if(holder.has_reagent(/datum/reagent/medicine/epinephrine))
 		holder.remove_reagent(/datum/reagent/medicine/epinephrine, 2 * REM * delta_time)
 	affected_mob.adjustPlasma(20 * REM * delta_time)
+	if(toxpwr && isplasmaman(affected_mob)) // plasma is not toxic to them
```

Comment:
You should add a `TRAIT_PLASMA_LOVER_METABOLISM` trait for plasmaman livers and move all the effects to it.  See how it's done for security liver:

https://github.com/tgstation/tgstation/blob/46594206f04eac960d6eb5baacc980e10df6e949/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm#L1006-L1014